crates/verify/src/provider.rs

```diff
@@ -171,6 +176,7 @@ impl VerificationProviderType {
             Self::Sourcify => Ok(Box::<SourcifyVerificationProvider>::default()),
             Self::Blockscout => Ok(Box::<EtherscanVerificationProvider>::default()),
             Self::Oklink => Ok(Box::<EtherscanVerificationProvider>::default()),
+            Self::Custom => Ok(Box::<EtherscanVerificationProvider>::default()),
```

Comment:
Could you add an extra comment detailing that `Custom` comes with the assumption of being compatible with the Etherscan API?

Reply:
added in https://github.com/foundry-rs/foundry/pull/9311/commits/3a9b873127b3915f4ad24c9006ee6476d525344f verifier args showing as

![image](https://github.com/user-attachments/assets/97187afe-7207-468f-95c5-3ee4a118f0f3)